quic/s2n-quic-core/src/recovery/pacing.rs

```diff
@@ -57,6 +57,12 @@ impl Pacer {
         max_datagram_size: u16,
         slow_start: bool,
     ) {
+        // low RTT networks should not using pacing since it'll take longer to wake up from
+        // a timer than it would deliver a packet
+        if rtt_estimator.smoothed_rtt() < Duration::from_millis(2) {
```

Comment:
I'm wondering if this check would be better located where the pacer is used in [connection_impl.rs](https://github.com/aws/s2n-quic/blob/main/quic/s2n-quic-transport/src/connection/connection_impl.rs#L874). We already have some degree of buffer there, since the `!edt.has_elapsed` check would not delay a packet if the pacer is recommending less than 1 ms delay.

Reply:
that's an option. but i kind of like it in the pacer itself as it's clear that our pacer implementation doesn't recommend pacing low RTT networks. if it's where it's used then it isn't entirely connected to the pacer code.

Reply:
I can agree with that. I was a little uneasy with how this would just not be updating `self.next_packet_departure_time` once the `smoothed_rtt` drops below 2ms, but it doesn't actually matter with our usage of the pacer. 

Can you make the `Duration::from_millis(2)` a const though?

Reply:
yeah for sure